### ISSUE TITLE
<scrip src="script-1.js"></scrip> does not work

### ISSUE THREAD

**Issue by Pomax** [mozilla/thimble.webmaker.org#1238]
The reason it doesn't work is pretty obvious, but there's nothing in Thimble that's telling me that I'm missing a fairly trivial "t" in the element tag.

**Comment by gideonthomas** [comment]
I think this calls for a "bump-up-in-priority" for fixing HTMLHinter

**Comment by flukeout** [comment]
@gideonthomas Is HTMLHinter a brackets plugin? Is it not picking up the error, or are we missing styles to help it display what's going on?

**Comment by gideonthomas** [comment]
@flukeout yep it is a brackets plugin but we have not enabled it on thimble because the UI is terrible.
You can try it by adding `?enableExtensions=HTMLHinter` to your url to see what it currently looks like

**Comment by flukeout** [comment]
Yikes! Haha...

![image](https://cloud.githubusercontent.com/assets/25212/11160009/cc752504-8a19-11e5-8ee8-a8cfb24d4a28.png)

**Comment by flukeout** [comment]
Also, will try out https://github.com/MiguelCastillo/Brackets-InteractiveLinter

**Comment by flukeout** [comment]
Made some progress with functionality. You can hover over the text in the error message to highlight the related code...

![html-lint](https://cloud.githubusercontent.com/assets/25212/23574858/9b73ed3c-0038-11e7-8dd2-4a287eba7af8.gif)

**Comment by humphd** [comment]
Getting https://github.com/MiguelCastillo/Brackets-InteractiveLinter to work likely involves re-working their logic to not try and lazy-load files from the filesystem, and instead use `require` to pre-load them.  I can try and get it working if you want.

One thing I was going to say about what you have above, and I see the other one does it already, is to make use of the gutter space beside the line numbers for the `!` icon, vs. having it float on the right.

This is all very cool!

**Comment by flukeout** [comment]
Having just answered a support email which ended up with me telling them to add a ";" to the end of one of the lines of CSS makes me think we should keep thinking about how to do this stuff.
